.git-blame-ignore-revs

```diff
@@ -0,0 +1,3 @@
+# This commit has completely rewritten the history of file `userspace/libscap/engine/savefile/scap_savefile.c`. 
+# https://github.com/falcosecurity/libs/commit/a6df8fc90e3be0209c5a5f9bb0443077cd5d0967
```

Comment:
I figured out that this commit completely overwrites the blame of userspace/libscap/engine/savefile/scap_savefile.c, so it seems reasonable to put it here